packages/angular/src/app/components/profile-card/profile-card.component.html

```diff
@@ -0,0 +1,39 @@
+<div class="panel">
+  <div class="title-text panel-header">
+    {{title}} </div>
+  <div class="form-container">
+    <ng-content select="[top-item]"></ng-content>
+    <dx-form [formData]="cardData"
+             [showColonAfterLabel]="true"
+             [colCount]="(screen.xSmallScreenChanged | async) ? 2 : colCount"
+             [screenByWidth]="getSizeQualifier"
+             (onFieldDataChanged)="dataChanged.emit($event)"
+             labelLocation="top"
+             labelMode="outside"
+    >
+      <ng-container *ngFor="let item of items">
+        <dxi-item [dataField]="item.dataField"
+                  [editorType]="item.editorType"
+                  [editorOptions]="assign | apply:{stylingMode: 'outlined'}: item.editorOptions"
+                  [colSpan]="item.colSpan">
+          <dxo-label *ngIf="item.label" [text]="item.label"></dxo-label>
+          <ng-container [ngSwitch]="item.dataField">
+            <ng-container *ngSwitchCase="'status'">
+              <status-select-box *dxTemplate
+                                 labelMode="hidden"
+                                 [(value)]="cardData[item.dataField]"></status-select-box>
+            </ng-container>
+
+            <ng-container *ngSwitchCase="'supervisor'"  >
```

Comment:
Let's remove redundant spaces:

```suggestion
            <ng-container *ngSwitchCase="'supervisor'">
```